Library/Formula/lighttpd.rb

```diff
@@ -25,10 +24,21 @@ class Lighttpd < Formula
   # default max. file descriptors; this option will be ignored if the server is not started as root
   MAX_FDS = 512
 
-  def config_path; etc+"lighttpd/"; end
-  def log_path; var+"log/lighttpd/"; end
-  def www_path; var+"www/"; end
-  def run_path; var+"lighttpd/"; end
+  def config_path
+    "#{etc}/lighttpd"
```

Comment:
I think we're getting this build error:
```
Error: undefined method `install' for "/usr/local/etc/lighttpd":String
```
Because by using interpolation you've changed `config_path` from a Pathname to a String. Try switching back to `etc/"lighttpd"` and see if it passes.